cluster_experiments/analytics/analysis_results.py

```diff
@@ -0,0 +1,109 @@
+from dataclasses import asdict, dataclass
+from typing import List
+
+import pandas as pd
+
+
+@dataclass
+class HypothesisTestResults:
```

Comment:
I'd avoid having two classes in here. One option is the following, though I'm not sure it's the best


Just HypothesisTestResults (consider renaming to ScorecardResults or Scorecard):
* it should have a list of metric_alias, a list of control_variant_name, a list of treatment_variant_name, etc as inputs.
* it should have a to_frame() / to_dataframe() attribute.
* it should have a `__add__` method.

The add method should append the lists of the two scorecards, the to_frame should materialize in a dataframe. wdyt?

In general, I'd like to have a single class to handle scorecards and an add method that allows us to stack them up, that's why I thought of this solution.